src/controller/java/zap-generated/chip/devicecontroller/ClusterInfoMapping.java

```diff
@@ -14634,4 +14636,1716 @@ public void combineCommand(
     readAttributeMap.put("windowCovering", readWindowCoveringInteractionInfo);
     return readAttributeMap;
   }
+
+  public Map<String, Map<String, InteractionInfo>> getWriteAttributeMap() {
```

Comment:
In other Java generated code, I've been splitting different functionalities into different files to avoid having one massive source file. Could we move this function into a new template `ClusterWriteMapping-java.zapt`, and then just call that statically here?